CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ project(
   DESCRIPTION
     "This template library is a collection of template-oriented code that we, the Data Science Group at UPB, found pretty handy. It contains: `switch_cases` (Use runtime values in compile-time context), `integral_template_tuple` (Create a tuple-like structure that instantiates a template for a range of values), `integral_template_variant` (A wrapper type for `std::variant` guarantees to only contain variants of the form `T<IX>` and `for_{types,values,range}` (Compile time for loops for types, values or ranges))."
   HOMEPAGE_URL "https://dice-research.org/")
-set(POBR_VERSION 1)  # Persisted Object Binary Representation
+set(POBR_VERSION 2)  # Persisted Object Binary Representation
```

Comment:
TODO figure out if we care that the layout of `polymorphic_allocator` changed. I would assume we don't

Reply:
Let's keep the version and remove polymorphic_allocator from the tracked files for POBR.